templates/topcrash_add_keyword.html

```diff
@@ -0,0 +1,27 @@
+<p>The following {{ plural('bug has', data, pword='bugs have') }} no has_str and a str in the first comment:
```

Comment:
Copy/paste side effect, I presume...

Reply:
Yes, fixed in c374a00476e286c02a3f2088ec040ac6e4c92b09.